backend/api/invoices/create/services/add.py

```diff
@@ -9,7 +9,7 @@
 @require_http_methods(["POST"])
 def add_service(request: HtmxHttpRequest):
     context: dict = {}
-    existing_service = request.POST.get("existing_service", "")
+    existing_service = request.POST.get("existing_service", 0)
```

Comment:
Just a bit confused why this should default to 0. POST.get will always return a string. I'm guessing in this case you want 0 to just represent None, but wouldn't it make more sense and be better to just use None? I'm confused how setting it to an integer fixes the errors lol

Reply:
oh yes, I just thought of the way to trigger exception and not a ValueError, None probably is a cleaner solution. I will change that since I found also few more issues of this type in that file. 
You want me to send them as seperate PR or together with `mypy` contribution docs update, CI, pre-commit hook ?

Reply:
Separate PR please, thanks. The smaller the PR the quicker I can merge. Otherwise it'll sit there for a month lol

Reply:
Hmm now as I have dug deeper into this `existing_service = request.POST.get("existing_service", 0)` the expected values by `existing_service_obj = InvoiceProduct.objects.get(user=request.user, id=existing_service)` are of a type `str | int` so even tho with `None` it works fine, in terms of type annotations `None` is a invalid value and `0` or `"0"` is a valid one. In this case I think that leaving it as `"0"` will be cleaner than placing `# type: ignore` since `product.id` never should be a `0`.